src/ServicePulse.Host/app/lib/bootstrap-3.3.5/css/bootstrap.css

```diff
@@ -1583,8 +1583,8 @@ pre code {
   overflow-y: scroll;
 }
 .container {
-  padding-right: 15px;
-  padding-left: 15px;
+  padding-right: 0;
+  padding-left: 0;
```

Comment:
We could add a custom class with those properties, but the same problem would remain. However, I don't see bootstrap 3.x breaking those properties. I also don't see a reason to upgrade bootstrap.